### PR TITLE
Refactor#1

### PR DESCRIPTION
**Pull request by maniknr-sureify**
frontend api response handled based on BE refactoring ,changed constants naming convention

### CODE REVIEW

**Review by harshavardhan-sureify — APPROVED** [review]

**Review by rameshguptha-sureify — APPROVED** [review]

File: src/components/admin/AllUsers.jsx [hunk 1]
```diff
@@ -2,300 +2,328 @@ import React, { useEffect, useState } from "react";
...
+                setLoading(false);
+            });
+    };
 
-  useEffect(() => {
-    fetchAllUsers();
-  }, []);
+    useEffect(() => {
+        fetchAllUsers();
+        // eslint-disable-next-line
```
Comment (abhishekdundigalla-sureify) [inline src/components/admin/AllUsers.jsx:132]: remove comments

File: src/components/admin/AllUsers.jsx [hunk 2]
```diff
@@ -2,300 +2,328 @@ import React, { useEffect, useState } from "react";
...
+            userData?.filter(
+                (user) =>
+                    user.name
+                        .toLowerCase()
+                        .includes(searchText.toLowerCase()) ||
+                    user.user_id.includes(searchText.trim())
+            )
+        );
+        setPage(0);
+        // eslint-disable-next-line
```
Comment (abhishekdundigalla-sureify) [inline src/components/admin/AllUsers.jsx:150]: remove comments

File: src/components/admin/PollAnswers.jsx [hunk 3]
```diff
@@ -36,6 +36,7 @@ export const PollAnswers = ({ pollId }) => {
     };
     useEffect(() => {
         fetchPollAnswers();
+        // eslint-disable-next-line
```
Comment (abhishekdundigalla-sureify) [inline src/components/admin/PollAnswers.jsx:39]: remove comments

File: src/components/admin/PollCreation.jsx [hunk 4]
```diff
@@ -94,6 +94,7 @@ const PollCreate = () => {
 
     useEffect(() => {
         if (isClicked) validateForm();
+        // eslint-disable-next-line
```
Comment (abhishekdundigalla-sureify) [inline src/components/admin/PollCreation.jsx:97]: remove comments

File: src/components/user/ViewPolls.jsx [hunk 5]
```diff
@@ -21,7 +21,9 @@ export const ViewPolls = ({ activeFlag, pollsData}) => {
                 ))
             ) : (
                 <EmptyDataContainer
-                    message="No polls found!!"/>
+                    message={message
+                    }
```
Comment (abhishekdundigalla-sureify) [inline src/components/user/ViewPolls.jsx:25]: align properly

File: src/components/user/ViewSinglePoll.jsx [hunk 6]
```diff
@@ -198,6 +206,7 @@ export const ViewSinglePoll = () => {
                                     mt: "15px",
                                 }}
                             >
+                                {/* <StyledBackIcon
                                 {/* <StyledBackIcon
                                     onClick={handleBackNavigation}
                                 /> */}
```
Comment (abhishekdundigalla-sureify) [inline src/components/user/ViewSinglePoll.jsx:212]: remove above three commented lines if not required

**Review by abhishekdundigalla-sureify — CHANGES_REQUESTED** [review]
please remove below line which is commented in several components
(eslint-disable-next-line)

File: src/components/user/ViewPolls.jsx (hunk 5 above)
Comment (maniknr-sureify) [reply src/components/user/ViewPolls.jsx]: done

File: src/components/user/ViewSinglePoll.jsx (hunk 6 above)
Comment (maniknr-sureify) [reply src/components/user/ViewSinglePoll.jsx]: done

**Review by abhishekdundigalla-sureify — APPROVED** [review]

**Pull request merged**